classes/ServerInterface.php

```diff
@@ -5,7 +5,17 @@
 require_once(MUMPHPI_MAINDIR.'/classes/HelperFunctions.php');
 require_once(MUMPHPI_MAINDIR.'/classes/MessageManager.php');
 
-if (extension_loaded('ice') && function_exists('Ice_intVersion') && Ice_intVersion() >= 30400) {
+if(!extension_loaded("ice"))
+{
+	printJson("IcePHP extension is not loaded. Revise your IcePHP installation.");
```

Comment:
Is printJson a valid function? I don’t think so?

Reply:
My mistake.  I had a test function and that was a left-over command.  Shall I submit a PR with that corrected?